File: app/src/main/java/com/supercilex/robotscouter/ui/scout/viewholder/HeaderViewHolder.java
package com.supercilex.robotscouter.ui.scout.viewholder;

import android.support.v7.widget.RecyclerView;
import android.view.View;

import com.supercilex.robotscouter.R;

public class HeaderViewHolder extends ScoutViewHolderBase {
    public HeaderViewHolder(View itemView) {
        super(itemView);
    }

    @Override
    protected void bind() {
        super.bind();
        if (getLayoutPosition() != 0) {
            RecyclerView.LayoutParams params = (RecyclerView.LayoutParams) itemView.getLayoutParams();
            params.topMargin =
                    (int) itemView.getResources().getDimension(R.dimen.card_padding_vertical);
        }
    }
}

File: app/src/main/java/com/supercilex/robotscouter/ui/scout/viewholder/HeaderViewHolder.kt
package com.supercilex.robotscouter.ui.scout.viewholder

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView
import com.supercilex.robotscouter.R

open class HeaderViewHolder(itemView: View) : ScoutViewHolderBase<String, TextView>(itemView) {
    override fun bind() {
        super.bind()
        if (layoutPosition != 0) {
            val params = itemView.layoutParams as RecyclerView.LayoutParams
            params.topMargin = itemView.resources.getDimension(R.dimen.list_item_padding_vertical_within).toInt()
        }
    }
}
